client/src/components/ListItems.js

```diff
@@ -2,14 +2,15 @@ import styled from "styled-components/macro";
 import PropTypes from "prop-types";
 import NavBar from "../components/NavBar";
 import FlipCard from "../components/FlipCard";
+import ErrorMessage from "../components/ErrorMessage";
 
 const BoxContainer = styled.div`
   display: flex;
   flex-direction: column;
   justify-content: center;
-  grid-area: content;
+  /* grid-area: content; */
   padding-top: 10%;
-  margin: 0 15%;
+  /* margin: 0 15%; */
   gap: 100px;
```

Comment:
If no longer needed, remove the comments ;)

Reply:
wow! thank you... you are the eagle eye i need :)